openmls/src/group/core_group/mod.rs

```diff
@@ -738,6 +747,56 @@ impl CoreGroup {
         }
     }
 
+    /// Store the given [`EncryptionKeyPair`]s in the `backend`'s key store
+    /// indexed by this group's [`GroupId`] and [`GroupEpoch`].
+    ///
+    /// Returns an error if access to the key store fails.
+    pub(super) fn store_epoch_keypairs<KeyStore: OpenMlsKeyStore>(
+        &self,
+        backend: &impl OpenMlsCryptoProvider<KeyStoreProvider = KeyStore>,
+        keypair_references: &[EncryptionKeyPair],
+    ) -> Result<(), KeyStore::Error> {
+        backend.key_store().store_epoch_keys(
+            // Retrieving our identity should not fail.
+            self.own_identity().unwrap_or(&[]),
```

Comment:
This looks like a footgun to me where we use the same default identity for different things if this fails.
Maybe `own_identity` shouldn't be an option. But we could at least add a `debug_assert` on this.

Also, you can't you use `unwrap_or_default`?